### PR TITLE
vimc-6490: Save intermediate files out in qs format

### PR DESCRIPTION
**Pull request by r-ash**
So that read & writes are quicker

Also means we can remove some of the logic which checks row formatting when we read the file back in.

1 question about this - is the csv format useful? Will researchers ever get this file locally and look at it in excel? Or will they always load it into R to explore it? If the latter qs seems like a good choice, if the first maybe keeping the file as csv makes sense

### CODE REVIEW

File: tests/testthat/test_stochastic_process.R [hunk 1]
```diff
@@ -511,9 +511,8 @@ test_that("Stochastic - with upload", {
 
   result$cal$deaths_pies <- round(result$cal$deaths_pies / 2)
 
-  new_csv_file <- tempfile(fileext = ".csv")
-  write.csv(x = result$cal, file = new_csv_file,
-            row.names = FALSE)
+  new_csv_file <- tempfile(fileext = ".qs")
```
Comment (richfitz) [inline tests/testthat/test_stochastic_process.R]: new_qs_file ?

**Review by richfitz — APPROVED** [review]
LGTM. I vote for no csv

**Comment by r-ash** [comment]
Made a further update to support running stochastic upload with both qs files and csv files (for supporting re-uploading old outputs should we need to tear down annex at some point)

File: R/stochastic_upload.R [hunk 2]
```diff
@@ -175,3 +148,56 @@ stone_stochastic_upload <- function(file, con, annex, modelling_group,
...
   invisible()
 }
+
+read_processed_stochastic_data <- function(file, is_cohort) {
+  type <- tools::file_ext(file)
+
+  if (type == "csv") {
+    read_stochastic_csv(file, is_cohort)
+  } else if (type == "qs") {
+    read <-read_stochastic_qs(file, is_cohort)
```
Comment (weshinsley) [inline R/stochastic_upload.R]: Is the `read <-` intentional?

**Review by weshinsley — APPROVED** [review]

File: R/stochastic_upload.R (hunk 2 above)
Comment (r-ash) [reply R/stochastic_upload.R]: Ah no, good spot

**Pull request merged**